cmake/external/zlib.cmake

```diff
@@ -0,0 +1,47 @@
+include (ExternalProject)
+
+set(zlib_INCLUDE_DIR ${CMAKE_CURRENT_BINARY_DIR}/external/zlib_archive)
```

Comment:
Do we need zlib? If I understand it correctly, this is only used as a downstream dependency for protobuf right?

Reply:
thank you! Yangqing. We don't need it. Removed.